cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml

```diff
@@ -0,0 +1,21 @@
+name: aks-multi-model
+type: online
+target: azureml:aks-gpu-deploy
+auth_mode: Key
+traffic:
+    etblue: 100
+deployments:
+    - name: etblue
+      model: azureml:multi-models:2
+      code_configuration:
+          code:
+             local_path: ./code
+          scoring_script: dummy.py
+      environment:
+            name: aks-multi-env
```

Comment:
Hi: is there a version for MIR v2? would suggest defaulting to MIR and have a comment for aks in the same yaml file.

Reply:
Hi Srini,
We have updated the yml files as below -
'aks-multi-env' to 'multi-env' 
'mir-multi-env' to 'multi-env'.

Please review and let us know for any updates.

Reply:
Thanks